### DISCUSSION TITLE
THANK YOU SO MUCH!

### DISCUSSION THREAD

**Discussion by xxxFRIARxxx** [Rem0o/FanControl.Releases#1719] [General]
Thank you SO MUCH for creating this program!  After having issues with EVGA Precision X1 and MSI Afterburner silently closing on default-install for 3+ machines, I figured there was never going to be simple, easy software to control all my fans without crashing.

Then I found your program about a year ago, and it's been smooth sailing ever since.

PLEASE keep updating this program and lengthening the lifespan.  This is the ONLY fan control software I've ever found that JUST WORKS.  It DOESN'T silently close, leaving all your sensitive thermals to chance...it doesn't error out...it just works.

I have no money to buy you a coffee or anything to help support, so I wanted to write this thank you in lieu of that.

Thank you again from all the people sick of bloated, laggy, buggy, fan-control software!

@Rem0o

**Comment by Shadesss** [comment]
Also want to thank you @Rem0o, Wish I had known about this software before, would've ditched Asrock's poor app long long time ago.

**Comment by kklittich** [comment]
I’ll express my gratitude as well, as a mechanical engineer who deals with thermal systems a lot AND someone who is overly sensitive to sounds this has been amazingly helpful.

I sometimes play around with sound recording and now I have a ‘Silent’ profile I can switch to easily, then switch back to the still very quiet but more performant ‘Moderate’ profile.
